src/silx/gui/plot/PlotWidget.py

```diff
@@ -3472,6 +3477,15 @@ def onMouseLeaveWidget(self):
 
     # Interaction modes #
 
+    def interaction(self) -> PlotInteraction:
+        """Returns the interaction handler for this PlotWidget"""
+        return self._eventHandler
+
+    def __interactionChanged(self):
+        """Handle PlotInteraction updates"""
+        if self.__isInteractionSignalForwarded:
+            self.sigInteractiveModeChanged.emit(None)
```

Comment:
In case this results from a call to `setInteractiveMode`, do not forward the event